src/vs/workbench/contrib/positronPreview/browser/positronPreviewServiceImpl.ts

```diff
@@ -496,4 +501,35 @@ export class PositronPreviewService extends Disposable implements IPositronPrevi
 		// It's a localhost http or https URL; we can handle it in the viewer.
 		return true;
 	}
+
+	public async openEditor(uri: URI, title?: string): Promise<void> {
+		// Create and store webview overlay for editor
+		const previewId = `editorPreview.${PositronPreviewService._previewIdCounter++}`;
+		this._editors.set(previewId, {
+			uri: uri,
+			webview: this.createPreviewUrl(previewId, undefined, uri),
+			title: title
```

Comment:
```suggestion
			title: title || uri.authority || uri.path
```
This lets the title of the editor tab be more logical (to me at least.)